backend/middleware/EventValidation.js

```
@@ -0,0 +1,109 @@
+        if (!poster || !urlPattern.test(poster)) {
+            errors.push('Valid poster URL is required');
+        }
+
+        // Location validation
+        if (!location || location.trim().length < 3 || location.trim().length > 200) {
+            errors.push('Location must be between 3 and 200 characters');
+        }
+
+        // Seats validation
+        const seats = parseInt(seatsAvailable);
+        if (isNaN(seats) || seats < 1 || seats > 10000) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)